cmd/libsnap-confine-private/mount-opt.c

```diff
@@ -256,9 +256,10 @@ static const char *use_debug_build =
     "(disabled) use debug build to see details";
 #endif
 
-void sc_do_mount(const char *source, const char *target,
-		 const char *fs_type, unsigned long mountflags,
-		 const void *data)
+static bool sc_do_mount_ex(const char *source, const char *target,
+			   const char *fs_type,
+			   unsigned long mountflags, const void *data,
+			   bool optional)
```

Comment:
Didn't we have a recent PR by @zyga adding optional mounts as well?

I suppose the PR is packing unrelated content?

Reply:
Yeah, that is from zygas PR, I used it as the base for my work. It is merged now so the diff is much smaller.